packages/react-devtools-core/src/standalone.js

```diff
@@ -200,6 +200,14 @@ var DevtoolsUI = {
     return DevtoolsUI;
   },
 
+  setBrowserName(name) {
+    config.browserName = name;
+    if (panel) {
+      panel.props.browserName = name;
```

Comment:
We shouldn't mutate props like this. Do you have an alternative solution that re-renders panel with the right props?